Sources/Hummingbird/Configuration.swift

```diff
@@ -33,6 +33,8 @@ public struct ApplicationConfiguration: Sendable {
     public var backlog: Int
     /// Allows socket to be bound to an address that is already in use.
     public var reuseAddress: Bool
+    /// Maximum active connections
+    public let maxActiveConnections: Int?
```

Comment:
Since there are multiple ways to limit connections besides a hard cap, I'm a little reluctant to add such a property here

Reply:
This is still a draft and more a demonstration of how we can limit available connections. I was thinking of adding a `Delegate` generic parameter to the ChannelHandler which gave us a little more flexibility. eg
```swift
protocol AvailableConnectionsDelegate {
    func addConnection()
    func removeConnection()
    func IsConnectionAvailable() -> Bool
}
```

Reply:
That's roughly what I'm thinking too